homeassistant/components/met_eireann/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "config": {
+        "step": {
+            "user": {
+                "title": "Met Éireann",
+                "description": "Enter your location to use weather data from the Met Éireann Public Weather Forecast API",
+                "data": {
+                    "name": "Name",
+                    "latitude": "Latitude",
```

Comment:
May I suggest that you add some extra common keys to the config_flow strings, as discussed on #40578?
At least for "latitude" and "longitude".

Reply:
That's done, thanks for the suggestion!